sir_cake/seller/views.py

```diff
@@ -120,3 +118,29 @@ def orders(request):
                'items': pagination(request, orders_and_prices, 30)}
 
     return render(request, 'seller/orders.html', context)
+
+
+@user_is_seller
+def order_detail(request, pk):
+    try:
+        order = Order.objects.exclude(status=Order.Status.PREORDER).get(id=pk)
```

Comment:
I think this should be `Order.objects.get(id=pk, status=Order.Status.PREORDER)`?

Also, why is `status=Order.Status.PREORDER` needed? Do we only allow users to check details of preorders and don't allow to check them once paid, etc?

Reply:
I excluded preorders because the seller only needs to see finished (paid) orders :)